### ISSUE TITLE
Support for label/ref commands provided by zref

### ISSUE THREAD

**Issue by Yarakashi-Kikohshi** [James-Yu/LaTeX-Workshop#4261]
## Pre-checks*

- [x] The feature request has not been suggested in this repository.

## The Missed*

`\zlabel` provided by [zref][] does not have “label name completion” and “syntax highlighting” like other `\label`s.

[zref]: https://ctan.org/pkg/zref "CTAN: Package zref"

Additionally, `\ref` accepts variations of this such as `\zref`, but syntax highlighting for `\zref` with options (`\zref[option]{foo}`) does not work.

```latex
\label{foo}
\zlabel{foo}

\zref{foo}
\zref[option]{foo}
```

![image](https://github.com/James-Yu/LaTeX-Workshop/assets/75878492/6d2f27cb-1475-43e5-ab04-e3c61d94aaf5)

## Anything Else?

None.

## Version

- LaTeX Workshop: 9.20.1
- VS Code 
    ``` 
    Version: 1.89.0 (user setup)
    Commit: b58957e67ee1e712cebf466b995adf4c5307b2bd
    Date: 2024-05-01T02:09:22.859Z
    Electron: 28.2.8
    ElectronBuildId: 27744544
    Chromium: 120.0.6099.291
    Node.js: 18.18.2
    V8: 12.0.267.19-electron.0
    OS: Windows_NT x64 10.0.22631
    ```

**Comment by Yarakashi-Kikohshi** [comment]
I apologize for my misleading and ambiguous explanation.

> I confirm that neither `\label` nor `\zlabel` have label name auto-completion. Can you elaborate on your request?



By “label name completion” I was referring to the meaning that labeling by `\zlabel` is not reflected in the auto-completion of labels in `\ref`.




In other words, when you use `\zlabel{foo}`, `foo` is not displayed as a candidate in `\ref`.
Since `\zlabel` is a command for cross-referenced labels like `\label`, I would like to see labels by `\zlabel` also displayed as candidates in `\ref`.

> Are you requesting a same colorization with `\ref{b}`?

Yes.

I think that `\zref` and `\zlabel` with options should be the same color syntax highlighting as `\ref` and `\label`.
I will submit this as an issue to vscode-latex-basic later.

**Comment by James-Yu** [comment]
You can use `latex-workshop.intellisense.label.command` to include `zlabel`.

For highlighting, please create an issue in the other repo.

**Comment by Yarakashi-Kikohshi** [comment]
I thought I had some grasp of this extension, but I did not know about this feature. I should have read the wiki more before posting.



In response to your comment, I have added `zlabel` as follows.

```json
  "latex-workshop.intellisense.label.command": [
    "label",
    "zlabel",
    "linelabel"
  ]
```



However, the configuration of this setting may not be reflected. Am I doing something wrong?

```latex
\documentclass{article}
\usepackage{zref}
\begin{document}

\begin{figure}
  FOO
  \caption{foo}
  \label{fig:foo}
\end{figure}

\begin{figure}
  BAR
  \caption{bar}
  \zlabel{fig:bar}
\end{figure}

\ref{fig:}

\end{document}
```

![image](https://github.com/James-Yu/LaTeX-Workshop/assets/75878492/a298c8e5-39c3-4592-b053-bb43f70618c4)

**Comment by James-Yu** [comment]
If it won't work, try reload vscode.

<img width="1169" alt="image" src="https://github.com/James-Yu/LaTeX-Workshop/assets/4210342/b534e285-e4d7-4fa5-98af-12c61c836a97">

**Comment by Yarakashi-Kikohshi** [comment]
I reloaded many times and the result did not change, but when I added `zlabel` to `latex-workshop.view.outline.commands`, it worked! (I got the idea from your screenshot.)

Thank you very much for your kind attention. I apologize for the trouble I have caused you.

**Comment by James-Yu** [comment]
Right, I pushed another fix that influences this issue. Will go live in the next release.
